onnx/backend/test/runner/__init__.py

```diff
@@ -458,9 +479,14 @@ def run(test_self: Any, device: str, **kwargs) -> None:
                         output_file, ref_outputs, model.graph.output[i].type
                     )
                 outputs = list(prepared_model.run(inputs))
-                self.assert_similar_outputs(
-                    ref_outputs, outputs, rtol=model_test.rtol, atol=model_test.atol
-                )
+                try:
+                    self.assert_similar_outputs(
+                        ref_outputs, outputs, rtol=model_test.rtol, atol=model_test.atol
+                    )
+                except TypeError as e:
```

Comment:
Possible to cover this line?

Reply:
I give more information to assert_similar_outputs to have a better error message instead of catching it.